scaaml/capture/input_generators/attack_point_iterator.py

```diff
@@ -43,6 +44,11 @@ def get_generated_keys(self) -> List[str]:
         """
 
 
+class LengthIsInfiniteException(Exception):
+    """This exception get's called when a number is infinite
+      and wants to be represented by __len__ function."""
```

Comment:
"""This exception is raised when the `__len__` function is called on an infinite iterator."""

Reply:
How was the original comment resolved? One does not "call" an exception, one "raises" an exception. At least that is the language used by https://docs.python.org/3/tutorial/errors.html Is there a reason why the docstring is saying that "exception get's called"?

Reply:
Sorry I changed it in the wrong place and thought is was resolved because of that